test/index.test.js

```diff
@@ -370,18 +370,39 @@ describe('OktaReactNative', () => {
     });
   });
 
-  describe('signOutTest', () => {
+  describe('signOutTest succeeded.', () => {
     let mockSignOut;
 
     beforeEach(() => {
       mockSignOut = require('react-native').NativeModules.OktaSdkBridge.signOut;
     });
 
-    it('calls native sign out method', () => {
-      signOut();
-      expect(mockSignOut).toHaveBeenCalledTimes(1);
+    it('calls native sign out method', async () => {
```

Comment:
looks like the test case description is not matching the implementation now.

maybe it can be two cases here:
1. calls native method
2. resolves expected result